devices/gpu/nvidia/stats_test.go

```diff
@@ -447,9 +447,10 @@ func TestStatsForItem(t *testing.T) {
 			},
 			ExpectedResult: &device.DeviceStats{
 				Summary: &structs.StatValue{
-					Unit:            TemperatureUnit,
-					Desc:            TemperatureDesc,
-					IntNumeratorVal: helper.Int64ToPtr(1),
+					Unit:              MemoryStateUnit,
+					Desc:              MemoryStateDesc,
+					IntNumeratorVal:   helper.Int64ToPtr(1),
```

Comment:
Why did you make this change? It is valid to have no denominator

Reply:
This changes the expected results so that summary stat value is the memory value not the temprature unit.  The memory stat has denominator value and updated Unit/Desc values.

Reply:
Misread